frontend/src/components/namespace/CreateNamespaceButton.tsx

```diff
@@ -143,15 +144,17 @@ export default function CreateNamespaceButton() {
         </DialogContent>
         <DialogActions>
           <Button
-            aria-label="Cancel"
+            data-testid="create-namespace-dialog-cancel-button"
+            aria-label={t('translation|Cancel')}
```

Comment:
you don't need aria-label if the button already has a descriptive name

Reply:
I think it would still be helpful for accessibility

Reply:
aria-label is used when there's no visible text, here we have the same text inside the button already. it is not needed

Reply:
done!